esmvaltool/cmorizers/data/cmorizer.py

```diff
@@ -390,8 +394,9 @@ def _run_pyt_script(self, in_dir, out_dir, dataset, start, end):
 
 class DataCommand():
     """Download and format data to use with ESMValTool."""
+
     def __init__(self):
-        with open(datasets_file) as data:
+        with open(datasets_file, 'r', encoding='utf8') as data:
```

Comment:
@bouweandela indeed, I have now added a bog standard encoding that yaml ingests :+1: